app-laptop/tuxedo-keyboard/tuxedo-keyboard-3.1.3-r1.ebuild

```diff
@@ -0,0 +1,24 @@
+# Copyright 1999-2022 Gentoo Authors
+# Distributed under the terms of the GNU General Public License v2
+
+EAPI=7
+CONFIG_CHECK="ACPI_WMI INPUT_SPARSEKMAP"
+
+inherit linux-mod toolchain-funcs
+
+DESCRIPTION="Kernel Module for Tuxedo Keyboard"
+HOMEPAGE="https://github.com/tuxedocomputers/tuxedo-keyboard"
+SRC_URI="https://github.com/tuxedocomputers/${PN}/archive/v${PV}.tar.gz -> ${P}.tar.gz"
+
+LICENSE="GPL-3"
+SLOT="0"
+KEYWORDS="~amd64 ~x86"
+IUSE=""
```

Comment:
Drop it

Reply:
Done

Reply:
I dropped, but QA CI complains on LICENSING. So need to bring it back.

Reply:
"SLOT="0" is a mandatory variable according https://devmanual.gentoo.org/ebuild-writing/variables/index.html

Reply:
You have to drop only the empty IUSE